core/src/test/java/org/apache/spark/JavaAPISuite.java

```diff
@@ -63,6 +63,7 @@
 // The test suite itself is Serializable so that anonymous Function implementations can be
 // serialized, as an alternative to converting these anonymous classes to static inner classes;
 // see http://stackoverflow.com/questions/758570/.
+@SuppressWarnings("serial")
```

Comment:
Ok, I'll but there the id instead of warning.

There's a lot of warnings when you run maven build. But I believe they were not shown by default when you clicked "import maven project" in fresh installation of scala-ide two month ago. Only these serialization warnings. Fixing more warnings would be for some other PR.